### PR TITLE
remove alertmanager from module

### LINKED ISSUE #40: Extract AlertManager
@dhavalshah02 suggested that we extract alertmanager into its own module so it can be composed together like we do with Grafana.

WDYT?

### PR DESCRIPTION
**Pull request by jdelucaa**
Closes #40 

- Removed Alertmanager components from this module and moved them to a [new module](https://github.com/philips-labs/terraform-cloudfoundry-alertmanager) in philips-labs.

Please give me your feedback on the new module too. 😄

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -2,7 +2,7 @@ locals {
   thanos_routes = var.thanos_public_endpoints ? [cloudfoundry_route.thanos.id, cloudfoundry_route.thanos_internal.id] : [cloudfoundry_route.thanos_internal.id]
   postfix       = var.name_postfix != "" ? var.name_postfix : random_id.id.hex
   prometheus_config = templatefile("${path.module}/templates/prometheus.yml", {
-    alertmanagers = ["0.tf-alertmanager-${local.postfix}.apps.internal:9093"]
```
[the remark this answers is not in the document]
Comment (ScottGuymer) [reply main.tf:5]: the leading `0.` indicates a specific instance on the internal routing on CF.

I think this has been done on purpose to make sure it will always go to a single instance..  @loafoe is that correct?

[the remark this answers is not in the document]
Comment (loafoe) [reply main.tf:5]: @jdelucaa @ScottGuymer correct. Promethues states there should be no load balancing in front of alertmanager. So once the alertmanager module gains that capability it should export the individual instances. Safest is even to deploy distinct alertmanager apps and have them clustered. Promethues should then have distinct routes to each of them.

TL;DR the `0.` is signifcant

**Comment by loafoe** [comment]
@jdelucaa nice work! We should publish the alertmanager module first before merging this

**Comment by jdelucaa** [comment]
> @jdelucaa nice work! We should publish the alertmanager module first before merging this

@loafoe Sure, I am wondering if I should already publish a first version with the ms teams embedded in there or extract that stuff out first.

**Review by ScottGuymer — APPROVED** [review]
